incubator/webpagetest-agent/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
Please use namespaced templates. If you create a new chart with `helm create` and Helm 2.7+ you'll see what this looks like.